src/writer/vote.ts

```diff
@@ -4,6 +4,19 @@ import { jsonParse } from '../helpers/utils';
 import { getProposal } from '../helpers/adapters/mysql';
 import db from '../helpers/mysql';
 
+async function getVp(body, msg, proposal) {
+  const scores = await snapshot.utils.getScores(
+    msg.space,
+    jsonParse(proposal.strategies),
+    proposal.network,
+    [body.address],
+    proposal.snapshot
+  );
+  return scores
+    .map((score: any) => Object.values(score).reduce((a, b: any) => a + b, 0))
+    .reduce((a, b: any) => a + b, 0);
```

Comment:
Since there's only one address, I find this a bit more readable:

```suggestion
  return scores
    .map((score: any) => Object.values(score)[0])
    .reduce((a, b: any) => a + b, 0);
```